components/dashboard/src/Login.tsx

```diff
@@ -209,7 +209,7 @@ export function Login() {
                 </div>
                 <div className="flex-none mx-auto h-20 text-center">
                     <span className="text-gray-400">
-                        By signing in, you agree to our <a className="gp-link hover:text-gray-600" target="gitpod-terms" href="https://www.gitpod.io/terms/">terms of service</a>.
+                        By signing in, you agree to our <a className="gp-link hover:text-gray-600" target="gitpod-terms" href="https://www.gitpod.io/terms/">terms of service</a> and <a className="gp-link hover:text-gray-600" target="gitpod-privacy" href="https://www.gitpod.io/privacy/">privacy policy</a>.
```

Comment:
**praise**: Looks pretty straightforward and also wraps nicely on viewports with smaller width.